### PR TITLE
Serve script documentation

### PR DESCRIPTION
**Pull request by gemfarmer**
⚠️ WIP

For #1914 

@mugizico I forgot that I had started doing this work. Here it is for reference.

### CODE REVIEW

**Comment by mugizico** [comment]
running `./serve` builds but returns  `ERROR '/assets/js/search-bundle.js' not found.` Per @gemfarmer, this is related to `jekyll_pages_search_api`

**Comment by mugizico** [comment]
I made a few edits on the `Contributing.md` in the last commit. I am also going to chat with someone from the [Writing Lab](https://github.com/18F/writing-lab) to see if it can be further improved upon.

**Comment by gemfarmer** [comment]
@mugizico Love the idea of talking with the Writing Lab!

**Comment by mugizico** [comment]
Thanks @gemfarmer! turns out since @coreycaitlin is already part of our team, she will be looking it over for us.

**Comment by coreycaitlin** [comment]
My commit updates the documentation trilogy: `README.md`, `CONTRIBUTING.md`, and `LICENSE.md`. Changes include:

README:

- Minor updates to clarify the current status of the repo, and move beta and prior-site links to a "History" section
- Move "Protected branches" information to the CONTRIBUTING file
- Remove IE version specification because it's redundant with CONTRIBUTING

CONTRIBUTING:

- Add a table of contents for easier navigation through this very long file
- Add a "Public contributions" section and move most content into a "18F contributions" section, as those are very different use-cases
- Clarify "Branches" section to include information about protected branches, as well as documentation of the two processes we'll be using for blog vs. development changes
- Add mention of where to find us in Slack
- Move linter information into the "Front end architecture" intro and mention Hound and Circle
- Minor style edits and some revisions in the "Front end architecture" section, based on conversation with @msecret 
- Add "Standards and benchmarks" header to clarify the scope of the subsections
- Add link to LICENSE in two places

LICENSE:

In reviewing this, I spotted some inaccuracies, and worked with @brittag to fix them. This file now:

- Clarifies that not all of the site is in the public domain, because it is based on WDS, which has a mix of licensing situations
- Links to WDS and notes which parts of the site have non-public-domain assets (for instance, fonts and images)
- Adds section for contributors

Definitely want review from @gboone and/or @elainekamlley before merging this.

**Comment by coreycaitlin** [comment]
Note: These files will need further refinement, but these were big enough changes that I wanted to get them reviewed and discussed before heading too much further down this path.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,61 +1,91 @@
...
-- The Sass code should be linted with `scss-lint` (see #1878)
-  - The [18F CSS linting configuration](https://raw.githubusercontent.com/18F/frontend/18f-pages-staging/.scss-lint.yml) will be used.
-  - If linting fails, it will also fail the tests, but not the build.
-- Will default to [semantic HTML5](http://www.w3schools.com/html/html5_semantic_elements.asp).
-
-
-### Images
-- Will use `<svg>` and `xlink` (looking for link) for icons.
-- All blog images should be under 800kb in total, un-minified size.
+This is a repo to design a beta version of 18f.gsa.gov using the [U.S. Web Design Standards](https://standards.usa.gov/) and to completely refresh most of the content.
```
Comment (gboone) [inline CONTRIBUTING.md]: Let's clarify this to indicate this repo is for 18f.gsa.gov, not a beta version. This is my bad, should have caught that before relaunching the site.

File: LICENSE.md [hunk 2]
```diff
@@ -1,32 +1,37 @@
-As a work of the United States Government, this project is in the
-public domain within the United States.
+This project includes a mix of the following:
 
-Additionally, we waive copyright and related rights in the work
-worldwide through the CC0 1.0 Universal public domain dedication.
+* Open source work that are not in the public domain
```
Comment (gboone) [inline LICENSE.md]: Should this be?
> "Open source works…"

File: README.md [hunk 3]
```diff
@@ -9,9 +13,9 @@ Run each of the following steps to get the site up and running.
 1. `git clone git@github.com:18F/18f.gsa.gov`
 2. `cd 18f.gsa.gov`
 3. `bundle install`
-4. `bundle exec jekyll serve`
+4. `./serve`
 
-To dramatically speed up rebuild times (from more than 20 seconds to around 5), you can also run `./serve` or `./build` for step 4. These are shorthand for a combination of commands that disable certain plugins. This is especially useful if you're drafting a blog post or formatting single pages. **With this method you will disable search and all archives.**
+To enable the ability to search and see archives, you can run `bundle exec jekyll serve` instead of `./serve` for the server start command. This will **slow down rebuild times dramatically**, so use this command with discretion.
```
Comment (gboone) [inline README.md]: Will users still have to clear out an environment variable when they switch from `./serve` to the standard `bundle exec` command?

Comment (coreycaitlin) [inline README.md]: I can't answer this one, sadly

Comment (mugizico) [inline README.md]: I switch between using `./serve` and `bundle exec jekyll serve` without clearing out an environment variable. Although, I am not sure if this is related to an earlier issue (before I joined this team)?

Comment (gboone) [inline README.md]: 🤔 Maybe this is a non-issue that I invented for myself. Never mind!

**Pull request merged**